roles/base/tasks/ansible-pull.yml

```diff
@@ -46,7 +46,7 @@
   git:
     repo: "{{ ansible_git_repository }}"
     dest: "/home/{{ service_account }}/repo"
-    version: master
```

Comment:
Need to switch this back to `master` for the merge.

Reply:
yes, the task is still WIP, I need to squash the hacks..